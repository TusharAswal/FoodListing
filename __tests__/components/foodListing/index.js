import React from 'react';
import FoodListing from 'components/foodListing';
import renderer from 'react-test-renderer';

test('renders correctly', async () => {
  const tree = renderer.create(<FoodListing />).toJSON();
  expect(tree).toMatchSnapshot();
});
