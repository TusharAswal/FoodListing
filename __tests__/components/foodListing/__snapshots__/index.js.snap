// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Modal
  animationType="fade"
  hardwareAccelerated={false}
  onRequestClose={[Function]}
  transparent={false}
  visible={true}
>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "backgroundColor": "#e9e8ee",
        "flex": 1,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "height": 107,
          "justifyContent": "center",
          "opacity": 1,
          "paddingHorizontal": 39,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "black",
              "fontSize": 49,
            },
            undefined,
            Object {
              "fontFamily": "FontAwesome",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
    <View
      style={
        Object {
          "height": 133,
          "justifyContent": "center",
          "paddingHorizontal": 39,
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 47,
            "fontWeight": "bold",
          }
        }
      >
        Approved Foods List
      </Text>
    </View>
    <View
      style={
        Object {
          "height": 133,
          "justifyContent": "center",
          "paddingHorizontal": 39,
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#D3D3D3",
            "borderRadius": 10,
            "flexDirection": "row",
            "height": 78,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flex": 0.1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "rgba(0,0,0,0.5)",
                  "fontSize": 29,
                },
                undefined,
                Object {
                  "fontFamily": "FontAwesome",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
        </View>
        <View
          style={
            Object {
              "flex": 0.9,
            }
          }
        >
          <TextInput
            allowFontScaling={true}
            onChangeText={[Function]}
            placeholder="Try searching fat, sauces, name..."
            rejectResponderTermination={true}
            style={
              Object {
                "height": "100%",
                "width": "100%",
              }
            }
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
      </View>
    </View>
    <RCTScrollView
      ListEmptyComponent={[Function]}
      contentContainerStyle={
        Object {
          "paddingHorizontal": 39,
        }
      }
      data={null}
      disableVirtualization={false}
      getItem={[Function]}
      getItemCount={[Function]}
      horizontal={false}
      initialNumToRender={10}
      keyExtractor={[Function]}
      maxToRenderPerBatch={10}
      onContentSizeChange={[Function]}
      onEndReachedThreshold={2}
      onLayout={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      showsVerticalScrollIndicator={false}
      stickyHeaderIndices={Array []}
      updateCellsBatchingPeriod={50}
      viewabilityConfigCallbackPairs={Array []}
      windowSize={21}
    >
      <View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 49,
                },
                undefined,
                Object {
                  "fontFamily": "FontAwesome",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
          <Text
            style={
              Object {
                "fontSize": 35,
                "paddingTop": 20,
              }
            }
          >
            No data available.
          </Text>
        </View>
      </View>
    </RCTScrollView>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#339af0",
          "borderRadius": 196,
          "height": 98,
          "justifyContent": "center",
          "position": "absolute",
          "right": 20,
          "top": 39,
          "width": 98,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#ffffff",
              "fontSize": 39,
            },
            undefined,
            Object {
              "fontFamily": "FontAwesome",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
  </RCTSafeAreaView>
</Modal>
`;
